src/repl/utils/gist.js

```diff
@@ -51,7 +57,14 @@ export function saveComponentAsGist ( source, json ) {
 		}
 	});
 
-	return fetch( `https://api.github.com/gists`, { method: 'POST', body })
-		.then( r => r.json() )
+	return new Promise( ( resolve, reject ) => {
+		const request = new XMLHttpRequest();
+		request.withCredentials = true;
```

Comment:
I wasn't clear on the intent of where the gist is created, but I assumed that credentials would be needed.